essentials.sh

```diff
@@ -185,6 +185,28 @@ def_protected_var() {
     >&2 echo "+ ${var_name}=*****"
 }
 
+#Defines variable assigning the specified value to it if the variable does not
+# exist for private variable (Test_descripton!)
+#
+# Globals:
+#     None
+# Arguments:
+#     Variable name
+#     Value by default
+# Returns:
+#     None
+def_priv_var() {
+    local var=$1
+    local val=$2
+
+    if check_if_variable_is_set ${var}; then
```

Comment:
I guess the check is redundant for private params. The function must be very simple. The main task is wrapping declaring variables -- there's no need to make it do more.